### PR TITLE
Feature/vue 68 adapt spacings

### PR DESCRIPTION
**Pull request by dominikbruechner**
adapted spacing to suit designers' needs

### CODE REVIEW

File: src/themes/default/styles/utility/_spacing.scss [hunk 1]
```diff
@@ -33,9 +33,9 @@
 }
 
 .mb--xl {
-	margin-bottom: var(--space-xl);
+	margin-bottom: var(--space-xxxl);
 }
 
 .mb--xxl {
-	margin-bottom: var(--space-xxl);
+	margin-bottom: var(--space-xxxxl);
```
Comment (adrianjost) [inline src/themes/default/styles/utility/_spacing.scss]: this does not look logical
class xxl creates a spacing of xxxxl?

File: src/themes/default/styles/variables/_spacing.scss [hunk 2]
```diff
@@ -6,7 +6,9 @@
...
 	--space-md: 1rem;
 	--space-lg: 1.5rem;
-	--space-xl: 3.25rem;
-	--space-xxl: 5.25rem;
-	--space-xxxl: 8.5rem;
+	--space-xl: 2rem;
+	--space-xxl: 2.5rem;
+	--space-xxxl: 3.25rem;
+	--space-xxxxl: 5.25rem;
+	--space-xxxxxl: 8.5rem;
```
Comment (adrianjost) [inline src/themes/default/styles/variables/_spacing.scss]: These are a lot of x''es. I don't think is is any usefull. We should find a better naming scheme for spacing. It's already hard to count the number of x'es

File: src/themes/default/styles/utility/_spacing.scss (hunk 1 above)
Comment (dominikbruechner) [reply src/themes/default/styles/utility/_spacing.scss]: oh right, it was just a plain find and replace, good call

File: src/themes/default/styles/variables/_spacing.scss (hunk 2 above)
Comment (dominikbruechner) [reply src/themes/default/styles/variables/_spacing.scss]: we should talk to Mona if this makes sense for her, and what naming would suit her, as we need to talk the same language :)

Comment (dominikbruechner) [reply src/themes/default/styles/variables/_spacing.scss]: and there are definitely too many x's, I got blurry vision just now from it

Comment (dominikbruechner) [reply src/themes/default/styles/variables/_spacing.scss]: changed and resolved

**Pull request merged**